vendor/modules.txt

```diff
@@ -919,6 +919,10 @@ github.com/keybase/go-crypto/rsa
 github.com/konsorten/go-windows-terminal-sequences
 # github.com/kr/fs v0.1.0
 github.com/kr/fs
+# github.com/kr/pretty v0.1.0
+github.com/kr/pretty
+# github.com/kr/text v0.1.0
+github.com/kr/text
```

Comment:
Not sure if this is anything to be concerned about. I'm guessing `kr` is a transitive dependence of `govmomi`?

Reply:
looks like it
```
Gopkg.lock:  name = "github.com/kr/pretty"
Gopkg.lock:    "github.com/kr/pretty",
Gopkg.toml:  name = "github.com/kr/pretty"
go.mod:replace github.com/kr/pretty v0.1.0 => github.com/dougm/pretty v0.0.0-20171025230240-2ee9d7453c02
go.mod: github.com/kr/pretty v0.1.0
govc/flags/output.go:   "github.com/kr/pretty"
vendor/github.com/kr/pretty/Readme:    import "github.com/kr/pretty"
vendor/github.com/kr/pretty/Readme:    http://godoc.org/github.com/kr/pretty
vendor/modules.txt:# github.com/kr/pretty v0.1.0 => github.com/dougm/pretty v0.0.0-20171025230240-2ee9d7453c02
vendor/modules.txt:github.com/kr/pretty
```